test/timearray.jl

```diff
@@ -494,4 +504,20 @@ end
 end  # @testset "show methods don't throw errors"
 
 
+@testset "getproperty" begin
+    let
+        @test cl.Close  == cl[:Close]
+        @test_throws KeyError cl.NotFound
```

Comment:
The type of error should be consistent with what's used by DataFrames

Reply:
yeah, that's why I throw `KeyError`
```julia
julia> using DataFrames
                                                                                                                                                                             
julia> df = DataFrame()
0×0 DataFrame                                                                                                                                                                
                                                                                                                                                                             
                                                                                                                                                                             
julia> df[:A]
ERROR: KeyError: key :A not found                                                                                                                                            
Stacktrace:                                                                                                                                                                  
 [1] getindex at ./dict.jl:478 [inlined]                                                                                                                                     
 [2] getindex at /home/iblis/.julia/packages/DataFrames/1PqZ3/src/other/index.jl:123 [inlined]                                                                               
 [3] getindex(::DataFrame, ::Symbol) at /home/iblis/.julia/packages/DataFrames/1PqZ3/src/dataframe/dataframe.jl:264                                                          
 [4] top-level scope at none:0                                                                                                                                               
                                                                                                                                                                             
julia> df.A
ERROR: KeyError: key :A not found                                                                                                                                            
Stacktrace:                                                                                                                                                                  
 [1] getindex at ./dict.jl:478 [inlined]                                                                                                                                     
 [2] getindex at /home/iblis/.julia/packages/DataFrames/1PqZ3/src/other/index.jl:123 [inlined]                                                                               
 [3] getindex at /home/iblis/.julia/packages/DataFrames/1PqZ3/src/dataframe/dataframe.jl:264 [inlined]                                                                       
 [4] getproperty(::DataFrame, ::Symbol) at /home/iblis/.julia/packages/DataFrames/1PqZ3/src/abstractdataframe/abstractdataframe.jl:241
```

Reply:
Ah, I see. For some reason I thought DataFrames did something else. Sorry about that.